src/auro-carousel.js

```diff
@@ -131,14 +149,12 @@ export class AuroCarousel extends LitElement {
   /**
    * Internal function to generate the HTML for the icon to use.
    * @private
-   * @param {string} svgContent - The imported svg icon.
-   * @returns {TemplateResult} - The html template for the icon.
+   * @param {string} category - The category of the auro-icon.
+   * @param {string} name - The name of the auro-icon.
+   * @returns {TemplateResult} - The html template for the auro-icon.
    */
-  generateIconHtml(svgContent) {
-    const dom = new DOMParser().parseFromString(svgContent, 'text/html'),
-      svg = dom.body.firstChild;
-
-    return svg;
+  generateIconHtml(category, name) {
+    return html`<${this.iconTag} customColor category="${category}" name="${name}" slot="icon"></${this.iconTag}>`;
```

Comment:
I have seen this in another PR. We should not be using auro-icon when we know specifically what icons are being used in an element. This places the UI in a precarious scenario if there is an issue with the CDN. 

Please revert.

Reply:
We have decided that this work will move forward and we will work on creating our own CDN.